test/code-fixtures/apex/TwinFieldMapperTests.cls

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
I don't think we should use proprietary code in public tests, let's remove this and squash merge to remove the code from the repo.

Reply:
All of the other classes in that folder are test classes I pulled from CPQ. Should we remove those too?

Reply:
Ya, let's not leak stuff. There is a lot of public github.com code we can use as examples.